### ISSUE TITLE
UserWarning: Possible precision loss when converting from float64 to uint16

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#1773]
~~~
sct_flatten_sagittal -i t1.nii.gz -s t1_seg.nii.gz

--
Spinal Cord Toolbox (jca_batchProc/dea98c961553315202c90086fddedf923e99a768)
Running /Users/julien/code/sct/scripts/sct_flatten_sagittal.py -i t1.nii.gz -s t1_seg.nii.gz

Smooth centerline/segmentation...
.. Get center of mass of the centerline/segmentation...
.. Smoothing algo = hanning
.. Windows length = 50
/Users/julien/code/sct/python/lib/python2.7/site-packages/skimage/util/dtype.py:110: UserWarning: Possible precision loss when converting from float64 to uint16
  "%s to %s" % (dtypeobj_in, dtypeobj))

Done! To view results, type:
fsleyes t1.nii.gz t1_flatten.nii.gz &
~~~

**Comment by abtahizadeh** [comment]
Is this related to #1800 ?

**Comment by jcohenadad** [comment]
@abtahizadeh absolutely-- i think this issue can be closed

**Comment by jcohenadad** [comment]
fixed via #1800
